question-3/index.html

```diff
@@ -3,16 +3,48 @@
 
 <!DOCTYPE html>
 <html>
+  <head>
+    <meta charset="utf-8" />
+    <script>
+      function getParagraph(content) {
+        const paragraphNode = document.createElement("p");
+        const textNode = document.createTextNode(content);
+        paragraphNode.appendChild(textNode);
+        return paragraphNode;
+      }
 
-<head>
-  <meta charset=utf-8 />
-</head>
+      function getAttributes() {
+        const linkElement = document.getElementById("w3r");
+        const href = linkElement.getAttribute("href");
+        const hreflang = linkElement.getAttribute("hreflang");
+        const rel = linkElement.getAttribute("rel");
+        const target = linkElement.getAttribute("target");
+        const type = linkElement.getAttribute("type");
+        const bodyElement = document.getElementsByTagName("body")[0];
+        bodyElement.appendChild(getParagraph(`linkElement: ${linkElement}`));
```

Comment:
Should consider having an empty line after variable declarations have ended. It should make the code more readable, shouldn't it?

Reply:
I am following airbnb style guide. That doesn't seem to have an empty line after variable declaration